tests/unit/services/ApiService.spec.ts

```diff
@@ -64,50 +65,54 @@ function setEnvironment(isValidOpReturn: boolean, isValidPowPegAddress?: boolean
   }
 };
 
-describe('function: createPeginTx', () => {
+describe( 'Api Service:',() => {
+  // TODO: Remove the static variable assignment when workflow set .env file
+  ApiService.baseURL = 'https://2wp-api.testnet.rsk.co';
```

Comment:
does this mean the apiservice is hitting the actual url? if so, this wouldn't be correct we should be sniffing the httprequest and mocking the response rather than rely on live environments, this is a unit test.

Reply:
This mean that if I don't set the variable `baseURL` on line 70, it will be undefined because is static. And yes, I seems to be pointing the actual URL. However I left the `TODO` because the API request mock is out of this scope